src/ar.erl

```diff
@@ -183,7 +187,7 @@ parse(["disk_space", Size|Rest], O) ->
 parse(["load_mining_key", File|Rest], O)->
 	parse(Rest, O#opts { load_key = File });
 parse(["start_hash_list", IndepHash|Rest], O)->
-	parse(Rest, O#opts { start_hash_list = ar_util:decode(IndepHash) });
+	parse(Rest, O#opts { start_hash_list = IndepHash });
```

Comment:
`IndepHash` is a hash of the block hash list, so calling it `BHLHash` as later in the code would make sense. Also, the `ar_util:decode/1` should probably still be called here since this is the parsing function.